### ISSUE TITLE
Beginning work

### ISSUE THREAD

**Issue by androidacy-user** [Librefox/librefox-website#1]
Hi @intika
I need desired content/design for the potential website
Here's what I have so far, and the potential site will be stored here and synced with GitHub
https://drive.google.com/folderview?id=1-828Uf4FQailZ3RbdnFNs5oLCei6-hG-

_Sent from my TETRA using [FastHub](https://play.google.com/store/apps/details?id=com.fastaccess.github)_

**Comment by intika** [comment]
Thank you for that, normally you have access to the git (write) you can do modification directly over-here...  also @samuel8941 have already started a proposition for the website here https://github.com/samuel8941/Librefox/tree/master/website-template you could may be see with him...
Also @AnXh3L0 can help on this one if he want to :) 

The idea is to have something similar to what Firefox have but more simpler 

Example (modern and clean)
https://www.mozilla.org/en-US/firefox/new/
https://www.waterfoxproject.org/en-US/waterfox/new/?scene=1

**Comment by androidacy-user** [comment]
I sent a request to you to allow FastHub to access your org-its the app I use

**Comment by intika** [comment]
Also i just activated the github page for this project https://librefox.github.io/librefox-website/ and applied a random github template just as example... 

but first i think we should ask what's the role of everyone as @samuel8941 already started and may be @AnXh3L0 wants to participate.

**I think a simple question for getting organized is:**

What do you (@samuel8941 @AnXh3L0 @colbycdev) want to do exactly on the website? the whole thing? sections? translation? etc...

**Comment by intika** [comment]
> I sent a request to you to allow FastHub to access your org-its the app I use

it's approved

**Comment by androidacy-user** [comment]
Thanks! Just pushed the index

**Comment by androidacy-user** [comment]
@intika I'd like to focus on design, proofreading, and editing. Also, I'd like to expand into more pages (download, faq, support, etc) 
Also, if you'd like, I can host your site
For free(for now) you'll get librefox.innonetlife.com, and be hosted on my main server. I support static/dynamic content, ftp, https, etc. Not everyone gets this for free; in support of this project I'll give it for free. I'll keep it up-to-date with GitHub repo
If you want a custom domain, or more server power, email me

_Sent from my TETRA using [FastHub](https://play.google.com/store/apps/details?id=com.fastaccess.github)_

**Comment by intika** [comment]
i saw that, it's visible here https://librefox.github.io/librefox-website/index-template.html (i renamed it to template to keep github template visible on main address)

**Comment by intika** [comment]
thank you for the hosting proposition, i already have some servers it's not a problem... but for simplicity and edit purpose using github page for the moment is a good solution.

**Comment by intika** [comment]
Wait and see what every one wants... 

Also for the main first site (template) i think it's more efficient if only one person do it... then for editing everyone could do whatever needed... 

For the hosting proposition i already have some servers so it's not really a problem... i also can provide ftp/sftp access to librefox.org if needed, but for the moment i think github page is a good solution for every one to edit what need to be edited...

**Comment by colbycdev** [comment]
Sorry I edited your comments by accident @intika
Anyway, let me know if you change your mind

**Comment by androidacy-user** [comment]
How can i check to see if the page i uploaded here displays correctly?
Figured it out from earlier comment. Never mind

**Comment by intika** [comment]
@colbycdev great but you are going a little bit too fast lool :D may be wait for others to reply first and for @samuel8941 to upload his work to get credit for it ^^ the site is not urgent as the next version of librefox is not ready

**Comment by androidacy-user** [comment]
I uploaded all his content already
I think we need credits.html-to acknowledge @samuel8941 as well as others
And I'm bored. Might as well do something  

_Sent from my TETRA using [FastHub](https://play.google.com/store/apps/details?id=com.fastaccess.github)_

**Comment by AnXh3L0** [comment]
I will check what has been done already and try to help where's possible. I'd rather work on the template of the site alone but others seem to have started already, so I'll follow up with them. 

I like the Firefox website so I believe I can easily adapt it for our use.

**Comment by intika** [comment]
@AnXh3L0 yeah this is what i thought a single person doing the template is more efficient... for the moment we are a little bit far from something that look like 

this https://www.mozilla.org/en-US/firefox/new/
or this https://www.waterfoxproject.org/en-US/waterfox/new/?scene=1

Basically right now we have the default github template and a template from html5up.com, I saw that you have a good experience in web-designing so if it's easy for you to bring some magic with a nice template that would be awesome :D... 

But first wait and see what @samuel8941 want to do...

Also if i have the time i will add an additional template and we will decide together what to pic ;)

**Comment by intika** [comment]
But bear in mind we have the time to do this ;) it's not urgent and also thank you all for your contribution :)

**Comment by AnXh3L0** [comment]
Ok cool.
I also propose we keep the website on GitHub Pages. We won't need anything dynamic as everything can be easily done with a few lines of JS (like detecting the OS of the visitor and providing them with a download button for their platform).

**Comment by samuel8941** [comment]
@AnXh3L0 have a look at the basic [idea](https://github.com/samuel8941/Librefox/tree/master/website-template) and see whether you can work with that, feel free to ditch it entirely or use whatever looks relevant. It was more a proof-of-concept and to get a page up quickly for introduction to librefox, once ready.

**Comment by androidacy-user** [comment]
@samuel8941 For now, I just copied your files to this repo. I've made some changes as well 

_Sent from my TETRA using [FastHub](https://play.google.com/store/apps/details?id=com.fastaccess.github)_

**Comment by intika** [comment]
@AnXh3L0 Alright, you have carte blanche, take your time ;)

**Comment by intika** [comment]
@colbycdev if you have any ideas/proposition let us know ;)

**Comment by androidacy-user** [comment]
@AnXh3L0 At least initially, I agree with you. In the future, however, we made need more. I tried hosting on github myself and found it limited. It also pretty much eliminates a custom domain, which would lend some credibility to us in consumer minds

_Sent from my TETRA using [FastHub](https://play.google.com/store/apps/details?id=com.fastaccess.github)_

**Comment by AnXh3L0** [comment]
> It eliminates a custom domain, which would lend some credibility to us in consumer minds.

Um no, GitHub does allow custom domains and it provides SSL ootb with Let's Encrypt. This means we won't have to worry about anything, and seeing as @intika already owns the librefox.org domain iirc he can directly link it with GitHub Pages.

**Comment by intika** [comment]
> can directly link it with GitHub Pages.

yes, it will be done later on, it's not relevant right now

**Comment by androidacy-user** [comment]
At least from the current setup and my (short) experience with GitHub pages, the best you can do is a redirect. Maybe I just didn't dig deep enough into it.
Ironically, my server also uses Let's encrypt. Though that's for internal resources, my customer facing pages use Cloudflare  

_Sent from my TETRA using [FastHub](https://play.google.com/store/apps/details?id=com.fastaccess.github)_

**Comment by androidacy-user** [comment]
OK. This is getting ridiculous. I'm going to go set us up a Gitter. I'll let you know the name here

**Comment by intika** [comment]
Slow down buddy it's not needed right now :+1:

**Comment by androidacy-user** [comment]
25 comments one issue. We need another way to communicate

**Comment by androidacy-user** [comment]
Just sent through the request for Gitter access to the org @intika

**Comment by theel0ja** [comment]
> 25 comments one issue. We need another way to communicate

There's threads with way more comments :-)

**Comment by androidacy-user** [comment]
@theel0ja Doesn't mean we should do that. Im on a small screen, so I have to scroll to the bottom to see the replies.... It's started taking way to long to scroll 😁

_Sent from my TETRA using [FastHub](https://play.google.com/store/apps/details?id=com.fastaccess.github)_

**Comment by AnXh3L0** [comment]
Lol, I have had to scroll down on 800+ comments on some issues using FastHub-Libre, but ok :P

**Comment by androidacy-user** [comment]
@AnXh3L0 LOL, I use FastHub too!
Still, I'd prefer not to have to. I also feel like the "issues" should be for that-not a chat room for us 😂

_Sent from my TETRA using [FastHub](https://play.google.com/store/apps/details?id=com.fastaccess.github)_

**Comment by intika** [comment]
😁 lool one more, it's starting with a good mood that's a good thing :D

**Comment by androidacy-user** [comment]
@intika Oh, you're sooo funny😁

_Sent from my TETRA using [FastHub](https://play.google.com/store/apps/details?id=com.fastaccess.github)_

**Comment by intika** [comment]
Anyone attending [FOSDEM](https://fosdem.org/2019/) @ Brussels?

**Comment by androidacy-user** [comment]
@intika Little too far. I live in the USA. Wish I could, though. Looks amazing 

_Sent from my TETRA using [FastHub](https://play.google.com/store/apps/details?id=com.fastaccess.github)_

**Comment by AnXh3L0** [comment]
I will probably be there @intika , a 90% chance hehe. 
Third year in a row if so :+1:

**Comment by intika** [comment]
@colbycdev Ooookay i understant why you are so energetic you did not sleep right 😁??

**Comment by intika** [comment]
@AnXh3L0 ohh great i'll probably be there too

**Comment by androidacy-user** [comment]
@intika I honestly don't sleep much. My 3 hours gives me the energy of everyone else 8 hours😂

_Sent from my TETRA using [FastHub](https://play.google.com/store/apps/details?id=com.fastaccess.github)_

**Comment by colbycdev** [comment]
@intika Could you please grant org access for Gitter? Any more than the 40+ comments and my $40 phone may crash🤣

**Comment by intika** [comment]
@colbycdev You did open this issue you need to take responsibility for it 😂 lool

No seriously it's too soon for Gitter, it would just be an other layer to maintain etc... it's more efficient here, and we all have limited time to assign to the project...

**Comment by androidacy-user** [comment]
@intika When my phone explodes, I'm blaming you 😁 

_Sent from my TETRA using [FastHub](https://play.google.com/store/apps/details?id=com.fastaccess.github)_

**Comment by androidacy-user** [comment]
Let me know when i can take the non template page live

**Comment by atbanpc** [comment]
Should we think about changing the logo icon a bit? The one currently in images might be too close to the branded Firefox builds and Mozilla does protect its trademarks.

**Comment by intika** [comment]
I am working on the Logo... i will post it later on... the one on images was added by @samuel8941 just as example

**Comment by atbanpc** [comment]
That's great.
Also, can I commit a few changes and fixes for the site?

**Comment by androidacy-user** [comment]
@atbanpc What changes do you want to make? 

_Sent from my TETRA using [FastHub](https://play.google.com/store/apps/details?id=com.fastaccess.github)_

**Comment by atbanpc** [comment]
Fix spelling mistakes and weird spacing as well as add some images where available.

**Comment by intika** [comment]
@atbanpc do you want to be added to the team ? ... for the PR it's not really needed as @AnXh3L0 will bring later on other website.

**Comment by androidacy-user** [comment]
@atbanpc Go ahead. I don't have a problem with it, and I don't see why @intika would either. Just in case, though, make a copy of any older files as file.extension.old(for now). In case something goes wrong 

_Sent from my TETRA using [FastHub](https://play.google.com/store/apps/details?id=com.fastaccess.github)_

**Comment by intika** [comment]
@atbanpc for the PR it's not really needed as @AnXh3L0 will bring later an other website. so may be wait a little bit until we decide what would be the definitive website

**Comment by atbanpc** [comment]
@colbycdev We're working in Git so we should not need copies of older files?

@intika Alright, I'll hold them off for a bit.

**Comment by intika** [comment]
@atbanpc i guess it's left there for @AnXh3L0 to see what is already there but you have write access to the git, clean it if you want ;)

**Comment by androidacy-user** [comment]
@atbanpc I must confess, I've never quite figured out how to view older versions of a file. I've mostly visited from FastHub, though, so that might be it

_Sent from my TETRA using [FastHub](https://play.google.com/store/apps/details?id=com.fastaccess.github)_

**Comment by intika** [comment]
@colbycdev it's available on the file history or on the commit section

**Comment by colbycdev** [comment]
@intika I'm seeing neither, so I'm guessing it's a limitation of FastHub.
Guess I can always RDP over to my tor relay server(its nickname is innonetife). It's hosted on Google cloud, and I have xrdp and GNOME on it. 

_Sent from my TETRA using [FastHub](https://play.google.com/store/apps/details?id=com.fastaccess.github)_

**Comment by Atavic** [comment]
> The idea is to have something similar to what Firefox have but more simpler

See: https://protocol.mozilla.org/ 

...and the [demo pages](https://protocol.mozilla.org/demos/)

**Comment by androidacy-user** [comment]
I'm thinking a site like [this](https://protocol.mozilla.org/demos/navigation.html) except in English, obviously  

_Sent from my TETRA using [FastHub](https://play.google.com/store/apps/details?id=com.fastaccess.github)_

**Comment by intika** [comment]
That's cool i did not know about this, will have a look at it when i work on the site... its a little different from the main mozilla's site tho but probably have some interesting css

**Comment by theel0ja** [comment]
That design is really minimalistic and looks nice IMO

**Comment by dimqua** [comment]
I agree, the main Mozilla's website is too complicated for this quite small project.

**Comment by intika** [comment]
i was referring to the design not the content or making it extended or so...

**Comment by androidacy-user** [comment]
Some people check their Facebook. Developers check gitbub 😅 

_Sent from my TETRA using [FastHub](https://play.google.com/store/apps/details?id=com.fastaccess.github)_

**Comment by intika** [comment]
hahahaha 10000 TURE... it's a social site after all !
when you apply for a visa for the USA they ask you for your social username and github is listed with facebook/twitter etc.

**Comment by dimqua** [comment]
@intika Don't you think that complexity of the design should be correspond with content? I mean, if you are not going to add a lot of content, you are probably better off with more minimalistic template.

**Comment by intika** [comment]
@dimqua yes true, but we don't need to make an identical copy just something that look similar, the template done by @samuel8941 is similar and is not that complicated

**Comment by shreyasminocha** [comment]
What's the current status of work on the site? I skimmed through the conversation, but it's a bit difficult to follow. Is there a template we've committed to work on or is there scope for working for scratch? I'm willing to help regardless.

PS: Most librefox.github.io links mentioned here seem to be broken.

**Comment by AnXh3L0** [comment]
I will be working on the website and logo soon. Will meet with @intika at FOSDEM after a few days and discuss about details; will let you all know.

**Comment by intika** [comment]
@shreyasminocha i added you to the team, you have write access to this repo as others ;)

currently there is not much, we have a basic github template 
- https://librefox.github.io 

And a template done by @samuel8941 
- https://github.com/samuel8941/Librefox/tree/master/website-template

We should lunch something similar to Firefox/Waterfox's site just before the independent Librefox releases will be ready

This post https://github.com/intika/Librefox/issues/108 will be kept as infos & news about the overall progress... 

I will meet wirh @AnXh3L0 next weekend at FOSDEM any one attending FOSDEM is welcome to join us ;) (just let us know here :)) 

We agreed that it's more efficient if only one person develop the main starting site/template, once this will be done any one can contribute when needed ;)

**Comment by colbycdev** [comment]
I think at this point we do need more than just a template up....
I mean, we can always revert to the template if needed

**Comment by androidacy-user** [comment]
And +1 on the independent builds
I'll start looking into forking Firefox source, unless someone has something they want to do already 

_Sent from my TETRA using [FastHub](https://play.google.com/store/apps/details?id=com.fastaccess.github)_

**Comment by androidacy-user** [comment]
Either FastHub changed, or I no longer have write access. There is no edit option on files anymore @intika  

_Sent from my TETRA using [FastHub](https://play.google.com/store/apps/details?id=com.fastaccess.github)_

**Comment by intika** [comment]
@colbycdev Mozilla, contacted me to avoid copyright and trademarks violation, i still did not have a final answer from them until then i would prefer minimizing the edit and forks, they are watching ;)

**Comment by shreyasminocha** [comment]
@intika Anything new?

**Comment by DontUseGithub** [comment]
any updates on this?
